src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviPlexusIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Prevent CTCPs"); break;
+		case 'N': return __tr2qs("N: Network administrator"); break;
+		case 'U': return __tr2qs("U: Network service"); break;
+		case 'X': return __tr2qs("X: Recipient for new server introduction and split messages"); break;
+		case 'w': return __tr2qs("w: Recipient for server WALLOPS"); break;
+		case 'z': return __tr2qs("z: Recipient for oper WALLOPS"); break;
+		case 'y': return __tr2qs("y: Recipient for notices on WHOIS"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+		case 'q': return __tr2qs("q: Services administrator, can use SQUIT"); break;
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviOftcIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Recipient for far connect/quit notices"); break;
+		case 'P': return __tr2qs("P: Network service"); break;
+		case 'w': return __tr2qs("w: Recipient for server WALLOPS"); break;
+		case 'z': return __tr2qs("z: Recipient for oper WALLOPS"); break;
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviBahamutIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'A': return __tr2qs("A: Server administrator"); break;
+		case 'C': return __tr2qs("C: Only accept private messages from users who share common channels"); break;
+		case 'F': return __tr2qs("F: Can bypass the ircd's recvq throttling"); break;
+		case 'I': return __tr2qs("I: Hide oper status"); break;
+		case 'K': return __tr2qs("K: Can see U:lined kill messages"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Connected over SSL"); break;
+		case 'X': return __tr2qs("X: Squelch without notice"); break;
+		case 'a': return __tr2qs("a: Services administrator"); break;
+		case 'b': return __tr2qs("b: Can see CHATOPS notices"); break;
+		case 'e': return __tr2qs("e: Recipient for blocked DCC notices"); break;
+		case 'f': return __tr2qs("f: Recipient for flood warnings"); break;
+		case 'g': return __tr2qs("g: Recipient for GLOBOPS notices"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'j': return __tr2qs("j: Recipient for rejected client notices"); break;
+		case 'm': return __tr2qs("m: Recipient for spambot notices"); break;
+		case 'n': return __tr2qs("n: Recipient for routing notices"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 's': return __tr2qs("s: Recipient for server KILL messages"); break;
+		case 'x': return __tr2qs("x: Squelch with notice"); break;
+		case 'y': return __tr2qs("y: Can see certain information requests (e.g. /stats)"); break;
```

Comment:
Finally someone who can correctly use a **proper** abbreviation of  e.g.

:+1:  hahaha